.github/workflows/require_changelog.yml

```diff
@@ -20,20 +20,20 @@ jobs:
           paths: ^CHANGELOG.md
       - name: set defaults
         run: |
-          echo "::set-env name=FEATURE_OR_RELEASE_OR_HOTFIX::false"
+          echo "FEATURE_OR_RELEASE_OR_HOTFIX=false" >> $GITHUB_ENV
       - name: enable check if this is a feature pr
         if: startsWith(github.head_ref, 'feature')
         run: |
-          echo "::set-env name=FEATURE_OR_RELEASE_OR_HOTFIX::true"
+          echo "FEATURE_OR_RELEASE_OR_HOTFIX=true" >> $GITHUB_ENV
       - name: enable check if this is a release pr
         if: startsWith(github.head_ref, 'release')
         run: |
-          echo "::set-env name=FEATURE_OR_RELEASE_OR_HOTFIX::true"
+          echo "FEATURE_OR_RELEASE_OR_HOTFIX=true" >> $GITHUB_ENV
       - name: enable check if this is a hotfix pr
         if: startsWith(github.head_ref, 'hotfix')
         run: |
-          echo "::set-env name=FEATURE_OR_HOTFIX::true"
-      - name: Changelog updates have been found
+          echo "FEATURE_OR_RELEASE_OR_HOTFIX=true" >> $GITHUB_ENV
```

Comment:
Is this a different variable name by accident or was this broken before?

Reply:
That's actually a copy and paste error, since the variable name is different on server. However, I don't think that this is problematic, because it might only adds a check if the branch is named "release". I could update as well, as you prefer.

Reply:
The last changelog action update was not applied, which alerts missing changes in release branches too, as they require an update from unreleased to next version. @Parnswir